### PR TITLE
Decompose onnx functions from spec

### PR DESCRIPTION
**Pull request by philass**

### CODE REVIEW

File: src/Compiler/CompilerOptions.cpp [hunk 1]
```diff
@@ -241,6 +241,10 @@ llvm::cl::opt<std::string> reportHeapAfter("report-heap-after",
                    "<output-files-base-path>.heap.log"),
     llvm::cl::init(""), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::list<std::string> functionsToDecompose("functions-to-decompose",
```
[the remark this answers is not in the document]
Comment (philass) [reply src/Compiler/CompilerOptions.cpp:288]: Yes, this is how I imagined it.

**Comment by philass** [comment]
@AlexandreEichenberger could you have another look at this?

Are there any changes you would like?

File: src/Compiler/CompilerOptions.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlexandreEichenberger) [reply src/Compiler/CompilerOptions.cpp:288]: For some reasons, though we would have done the reverse; I assume that we will refine how to determine which onnx ops are inlined and which ones are not.

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM

File: src/Compiler/CompilerOptions.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (philass) [reply src/Compiler/CompilerOptions.cpp:288]: I agree. Landing this allows me to play around with this in the Groq codebase.

I'm hoping I can share my learning and refine this afterwards.

**Pull request merged**